requirements-tests.txt

```diff
@@ -10,3 +10,19 @@ coverage==3.7
 requests==2.0.1
 six==1.4.1
 waitress==0.8.7
+boto3==1.4.0
+boto==2.42.0
```

Comment:
moto (the S3 mock library) depends on boto. The whole Flask/Jinja/Werkzeug/etc. dependency chain is required by moto as well. A production setup should only require boto3 and botocore.